exporter/loadbalancingexporter/log_exporter_test.go

```diff
@@ -195,7 +195,7 @@ func TestConsumeLogsUnexpectedExporterType(t *testing.T) {
 
 	// verify
 	assert.Error(t, res)
-	assert.EqualError(t, res, fmt.Sprintf("unable to export logs, unexpected exporter type: expected *exporter.Logs but got %T", newNopMockExporter()))
```

Comment:
why this change? Something is off...

Reply:
this would have changed here: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16830/files?file-filters%5B%5D=.go&show-deleted-files=true&show-viewed-files=true